### PR TITLE
[MAD-13759] Update Wordpress to version 6.3

### PR DESCRIPTION
**Pull request by davemednick**

### CODE REVIEW

File: magento/2/Magento-CE-2.4.5.tar.gz.md5 [hunk 1]
```diff
@@ -1 +1 @@
-8d1deeb4ba594861840a9dc1c45d8ba8
+a1996e49ea40fbe84822fd48f4bf962f
```
Comment (nortoh) [inline magento/2/Magento-CE-2.4.5.tar.gz.md5]: Curious if there should be Magento related files in this PR

Comment (davemednick) [reply magento/2/Magento-CE-2.4.5.tar.gz.md5]: I honestly have no idea where those came from.. They were definitely not touched by me.

Comment (davemednick) [reply magento/2/Magento-CE-2.4.5.tar.gz.md5]: comes from `commit 21133007a0e3c6b8f594e6e105ad2bc173c3db86`

**Comment by lukecav** [comment]
WordPress 6.3 has been released.
https://wordpress.org/news/2023/08/lionel/

**Pull request closed without merging**